### PR TITLE
Stdlib typofix

### PR DESCRIPTION
**Pull request by tpltnt**
Hi there,

I fixed to minor issues on the stdlib introduction page.
Cheers,
tpltnt

### CODE REVIEW

File: www/website-content/pages/documentation/introduction/stdlib-introduction.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ Target Specific APIs:
 
 * [neko](http://api.haxe.org/neko/index.html):
 
-	* [neko.Lib](http://api.haxe.org/neko/Lib.html): Low level interactions with neko platform
+	* [neko.Lib](http://api.haxe.org/neko/Lib.html): Low level interactions withthe neko platform
```
Comment (nadako) [inline www/website-content/pages/documentation/introduction/stdlib-introduction.md]: Seems like a space missing here

**Comment by tpltnt** [comment]
@nadako I fixed it. Thank you for telling me.

**Comment by tpltnt** [comment]
ping

**Comment by markknol** [comment]
Thanks!

**Pull request merged**